content/learn/arns/ants.mdx

```diff
@@ -17,31 +16,48 @@ The controller can only manage the ANT and set and update records, name, and the
 
 Owners of names should ensure their ANT supports evolve ability if future modifications are desired. Loss of a private key for a permanently purchased name can result in the name being "bricked".
 
+### Under_name Ownership
+
+Undernames can have an `owner` set on them. This owner is empowered to set that undername as their primary name, can remove that undername as their primary name, and has full control over that Undername's metadata, such as:
+
+- transactionId
+- ttlSeconds
+- owner
+- description
+- displayName
+- keywords
+- logo
```

Comment:
Lets capitalize these - and add short description of what each means (e.g. `Transaction ID - the data the reocord resolves to`)